src/Simulation/Native/src/simulator/wavefunction.hpp

```diff
@@ -338,7 +427,7 @@ class Wavefunction
   private:
     unsigned num_qubits_;             // for convenience
     mutable WavefunctionStorage wfn_; // storing the wave function
-    std::vector<qubit_t> qubitmap_;   // mapping of logical to physical qubits
+    mutable std::vector<qubit_t> qubitmap_;   // mapping of logical to physical qubits
```

Comment:
Does reorderWFN have to be const?

Reply:
Yes as it is called inside flush() which is const